src/common/vectorclock.js

```diff
@@ -78,9 +101,7 @@ exports.isFirstImmediateToSecond = (first, second) => {
 exports.incrementAll = (_clock, authorClock) => {
```

Comment:
Would it make more sense to call this `sumAll`?

Reply:
Good idea. The idea behind my name was "incrementEachBy", but that's a better name.